Makefile

```diff
@@ -49,6 +50,7 @@ lint-python:
 	flake8 --max-complexity 10 --max-line-length 120 --ignore E203,W503 pydantic_core tests setup.py
 	$(isort) --check-only --df
 	$(black) --check --diff
+	$(pyright)
```

Comment:
I think easier to remove this here, just running `make` will call pyright anyway.

Reply:
I see what you mean now, removed